Fika.Dedicated/Models/DedicatedStatus.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Fika.Dedicated.Models
+{
+    public readonly struct DedicatedStatus
+    {
+        public static string READY = "ready";
```

Comment:
Why `string` and not `enum`?

Reply:
because the server currently needs a string "ready" or "inraid", not an integer value. i didnt feel like changing this at the moment.

Reply:
now using enum, done.